### DISCUSSION TITLE
Aliasing result field names in DAX studio?

### DISCUSSION THREAD

**Discussion by NoahLull** [DaxStudio/DaxStudio#1109] [General]
I love the feature to export the results to an excel file for my users. The results are prefixed to the table name which muddies up presentation for my users.  Is there a way to give the field name an alias, or to just remove the table name prefix?
Example: '02-Fiscal Calendar'[Accounting Date] --> [Date]

In SQL I can simply write it as 
Select [Accounting Date] as [Date] 
and would like to be able to write a similar alias for dax.
Thank you!

**Comment by dgosbell** [comment]
It depends on the query. Typically DAX Studio will try to "clean up" the column names and I only leave the table prefix if there are duplicate column names.

One approach in DAX is to wrap your table expression in the SELECTCOLUMNS function where you can alias the column names

So instead of the following which should return a `Class` and `Color` column
```
EVALUATE
SUMMARIZECOLUMNS (
    'Product'[Class],
    'Product'[Color]
)
```
You could use SELECTCOLUMNS to alias these as `Class2` and `Color2` 
```
EVALUATE
SELECTCOLUMNS (
    SUMMARIZECOLUMNS (
        'Product'[Class],
        'Product'[Color]
    ),
    "Class2", 'Product'[Class],
    "Color2", 'Product'[Color]
)
```

**Reply by NoahLull** [reply]
Thank you!
Was able to rewrite something like this and it worked! 

EVALUATE
SELECTCOLUMNS(

SUMMARIZECOLUMNS(
    'Calendar'[CalendarQuarter],
    Customers[Occupation],
    "Total Sales", [Total Sales]
),    "QTR",'Calendar'[CalendarQuarter],
    "OCC",Customers[Occupation],
    "Total Sales", [Total Sales])


ORDER BY 
    "QTR" ASC, "OCC" ASC
